SoftLayer/CLI/account/orders.py

```diff
@@ -0,0 +1,38 @@
+"""Order list account"""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer.managers.account import AccountManager as AccountManager
+from SoftLayer import utils
+
+
+@click.command()
+@click.option('--limit', '-l',
+              help='How many results to get in one api call, default is 100',
+              default=100,
+              show_default=True)
+@environment.pass_env
+def cli(env, limit):
+    """Order list account."""
+    manager = AccountManager(env.client)
+    orders = manager.get_account_all_billing_orders(limit)
+
+    table = []
+    order_table = formatting.Table(['id', 'State', 'user', 'PurchaseDate', 'orderTotalAmount', 'Items'],
```

Comment:
Change column name to 

`order_table = formatting.Table(['Id', 'State', 'User', 'Date', 'Amount', 'Item'],`